package.json

```
@@ -28,19 +30,26 @@
   "homepage": "https://github.com/mercurius-js/auth",
   "devDependencies": {
     "@sinonjs/fake-timers": "^7.0.5",
+    "@types/node": "^15.0.2",
+    "@types/ws": "^7.4.2",
     "autocannon": "^7.0.5",
-    "concurrently": "^6.0.2",
+    "concurrently": "^6.1.0",
     "fastify": "^3.0.2",
     "mercurius": "^7.5.0",
     "pre-commit": "^1.2.2",
     "snazzy": "^9.0.0",
-    "standard": "^16.0.3",
     "tap": "^15.0.2",
+    "ts-standardx": "^0.7.5",
+    "tsd": "^0.14.0",
+    "typescript": "^4.2.4",
     "wait-on": "^5.3.0"
   },
   "dependencies": {
     "fastify-error": "^0.3.0",
     "fastify-plugin": "^3.0.0",
     "graphql": "^15.4.0"
+  },
+  "tsd": {
+    "directory": "./test"
```

Please put into `test/types` folder

```suggestion
    "directory": "./test/types"
```